### PR TITLE
feat:added cart page

### PR DESCRIPTION
**Pull request by shrushti2000**
added cart page.

image

### CODE REVIEW

File: pages/cart/cart.html [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+                        </span>
+                    </div>
+                    <div class="card-footer flex-hz">
+                        <button class="btn btn-primary card__btn-primary">ADD TO CART</button>
+                        <button class="btn btn-outline-primary card__btn-secondary">REMOVE FROM WISHLIST</button>
+                    </div>
+                </div>
+            </div>
+            <div class="card__container card-horizontal flex-hz">
+                <img class="card__image img-hz" src="../../images/prd-img.jpg" />
```
Comment (chanchal-panpaliya) [inline pages/cart/cart.html:56]: alt??? The required alt attribute specifies an alternate text for an image, if the image cannot be displayed.

**Review by chanchal-panpaliya — APPROVED** [review]
code looking good to merge .

File: pages/cart/cart.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  margin: 1rem;
+}
+.cart-summary-container {
+  grid-area: cart-summary-container;
+  padding: 2rem;
+  width: 30rem;
+  border-radius: 5px;
+  height: 40rem;
+  background-color: var(--gray);
+  box-shadow: rgba(149, 157, 165, 0.2) 0px 8px 24px;
```
Comment (tushargoyal812) [inline pages/cart/cart.css:20]: suggestion: add color variables

File: pages/cart/cart.html [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+                            <p class="card-text">5</p><i class="fa fa-minus card-icon-plus"></i>
+                        </span>
+                    </div>
+                    <div class="card-footer flex-hz">
+                        <button class="btn btn-primary card__btn-primary">ADD TO CART</button>
+                        <button class="btn btn-outline-primary card__btn-secondary">REMOVE FROM WISHLIST</button>
+                    </div>
+                </div>
+            </div>
+            <div class="card__container card-horizontal flex-hz">
```
Comment (tushargoyal812) [inline pages/cart/cart.html:55]: use consistent class naming if you are using hyphen than use it everywhere.

**Review by tushargoyal812 — COMMENTED** [review]
take care of suggestions,looks good to merge

**Pull request merged**